releasenotes/notes/prepare-2.2.0-99d722b656fba0a8.yaml

```
@@ -0,0 +1,28 @@
+---
+prelude: >
+    The Qiskit v2.2 release further increases the functionality and versatility of the SDK
+    including enhancements for the C-API and the transpiler as well as many other improvements
```

```suggestion
    including enhancements for the C API and the transpiler as well as many other improvements
```
AFAIK the docs team prefer it this way.